ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventLRspec.json

```diff
@@ -535,5 +535,24 @@
         "CRUD": "R"
       }
     ]
+  },
```

Comment:
Can this be moved to a CUI related file please. So that we can toggle the feature on and off